src/styles/main.scss

```
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

```suggestion
@import "header";
@import "navigation";
@import "container";
@import "card";
@import "reset";
```
Don't add imports in one line. It's more clear when every import starts from a new line.